Workflow/Sources/SubtreeManager.swift

```diff
@@ -82,15 +82,19 @@ extension WorkflowNode {
             childWorkflows = context.usedChildWorkflows
             sideEffectLifetimes = context.usedSideEffectLifetimes
 
-            /// Captured the reusable sinks from this render pass.
+            /// Capture the reusable sinks from this render pass.
             previousSinks = context.sinkStore.usedSinks
 
             /// Capture all the pipes to be enabled after render completes.
             eventPipes = context.eventPipes
-            eventPipes.append(contentsOf: context.sinkStore.eventPipes)
+            for (_, sink) in context.sinkStore.usedSinks {
+                eventPipes.append(sink.eventPipe)
+            }
 
             /// Set all event pipes to `pending`.
-            eventPipes.forEach { $0.setPending() }
+            for eventPipe in eventPipes {
```

Comment:
not really a functional change here, but we use for-in everywhere else in this method so we might as well be consistent